src/features/journal/components/JournalContentForToday.tsx

```
@@ -0,0 +1,32 @@
+  return (
+    <div className="space-y-8">
+      <Separator className="my-10" />
+
+      {todaysEntries?.entries.map((entry, index) => {
+        return (
+          <div key={entry._creationTime} className="px-8">
+            <h2 className="mb-4 text-2xl">{entry.title}</h2>
+
+            <div
+              className="prose prose-sm mb-8 max-w-none dark:prose-invert"
+              dangerouslySetInnerHTML={{ __html: entry.content }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)